src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -0,0 +1,185 @@
+$pf-c-tree-view--MaxNesting: 10;
+
+.pf-c-tree-view {
+  --pf-c-tree-view--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-tree-view--PaddingBottom: var(--pf-global--spacer--sm);
```

Comment:
do we need this or should it come from the parent layout? This padding exists in the `pf-c-[dropdown-menus]__menu element`, which is where tree view would go, so you would have double padding.

Reply:
maybe a padding modifier to achieve this padding ?